hil_starter.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+hil_dir=$1
+
+if [ "$#" -ne 1 ] || ! [ -d "$1" ]; then
+    echo "Usage: $0 path_to_HIL_directory" >&2
+    exit 1
+fi
+
+start_hil_server(){
```

Comment:
It's a separate file and you can always delete it. If I move this script elsewhere it, probably, won't be used because other people won't see it.